src/azure-cli/azure/cli/command_modules/servicefabric/operations/managed_clusters.py

```diff
@@ -167,6 +167,12 @@ def add_client_cert(cmd,
         if not cluster.public_i_pv6_prefix_id:
             cluster.public_i_pv6_prefix_id = None
 
+        if not cluster.public_ip_prefix_id:
```

Comment:
this looks duplicated from the lines above

Reply:
No these are two different properties

Reply:
removed